api-testing/functional/tests/cypress/fixtures/token.json

```diff
@@ -0,0 +1,3 @@
+{
+  "access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJWZGp4Nlp0X0ZUbHcwbmhHMVFyVi0yeWI2cmI1M1R6Um5Ib0ZkV2cwWXAwIn0.eyJleHAiOjE2NTM1MDAzNzMsImlhdCI6MTY1MzUwMDA3MywianRpIjoiNWQxMGJhNzktYmVmOC00YTgzLWEyZDktNzI3MjI1MWM3NTA0IiwiaXNzIjoiaHR0cHM6Ly9kZXYub2lkYy5nb3YuYmMuY2EvYXV0aC9yZWFsbXMvanhvZTJvNDYiLCJhdWQiOiJzaW1zLWFwaSIsInN1YiI6IjkxYjFjZTdmLWYwZjQtNDZiZi1hMTMzLTljYjI1MzYxN2JmOSIsInR5cCI6IkJlYXJlciIsImF6cCI6Imluc3RpdHV0aW9uIiwic2Vzc2lvbl9zdGF0ZSI6Ijk2OTg1ZmI4LTY5ZjUtNGQwZi05NjYxLWY3ZDBlOWY5ZGEyMiIsImFjciI6IjEiLCJyZXNvdXJjZV9hY2Nlc3MiOnsiaW5zdGl0dXRpb24iOnsicm9sZXMiOlsiSW5zdGl0dXRpb25BZG1pbiJdfX0sInNjb3BlIjoiSURQIGRlZmF1bHQtbmFtZS1zY29wZSBzaW1zLWFwaS1hdWRpZW5jZS1zY29wZSB1c2VybmFtZSIsImxhc3ROYW1lIjoiTm90QXZhaWxhYmxlIiwiSURQIjoiQkNFSUQiLCJnaXZlbk5hbWVzIjoiRTJFIFVzZXIgVGVzdCBmb3IgQkNlSUQiLCJ1c2VyTmFtZSI6IjQ4NjFlOTIwODJiZDQ4ODQ4NDJmMzA0YzM5ZjYzZmNiQGJjZWlkIiwiaWRwX3VzZXJfbmFtZSI6InNpbXNfZTJldGVzdHMifQ.NZlIeX-oGJmpVgHYFYLg4DO-x695ri-UG0xhwD4E5LTRYD2K-BCwv0tKdiLlyEYYjnptVzDOcWdis3AJ40hl6LE-JDSf752ZJ0n9CfyJnGMupuNKDAioDiNN4CdMj1UvJ4BchR8GAH8IiOUXjlGoaBfi_q0FJg4K2kQ44IaXgoLmQ6Laxxq_SdX9c9JSjloPCVZTfSExC0LQEUFRL0mNoZdcQ2g08ZXuUytOM7_upjGW3HsM4KpYT01dTxwvhoFREH0WYVLyJmn2HSduMZIsS5S3yfAAxyuOZc8yWlE2Vh2l_UEiTsY8u47qttJr7WZNAwSUOeWhKFz467eN080Bpg"
```

Comment:
Is there a reason to keep the token here?

Reply:
Removed it from here, passing token as a parameter in api call.